docs/docs/cmd/spo/listitem/listitem-roleinheritance-break.md

```diff
@@ -0,0 +1,54 @@
+# spo listitem roleinheritance break
+
+Break inheritance of list item. Keeping existing permissions is the default behavior.
```

Comment:
Let's move the comment about default behavior to the Remarks section like we did here: https://github.com/pnp/cli-microsoft365/blob/99da27d7a47c48eae97db6a6631d4826fdf1a5de/docs/docs/cmd/spo/list/list-roleinheritance-break.md#remarks

Reply:
✅